crypto-ffi/bindings/jvm/src/main/kotlin/com/wire/crypto/CoreCryptoContext.kt

```diff
@@ -653,24 +679,54 @@ class CoreCryptoContext(private val cc: com.wire.crypto.uniffi.CoreCryptoContext
         }
     }
 
+    /**
+     * Proteus last resort prekey stuff
+     *
+     * @return A CBOR-serialize version of the PreKeyBundle associated with the last resort PreKey (holding the last resort prekey id)
```

Comment:
```suggestion
     * Create a new last resort prekey
     *
     * @return A CBOR-serialize version of the PreKeyBundle associated with the last resort PreKey (holding the last resort prekey id)
```

Reply:
Yeah, this one is terrible. I did simply copy it from the JS bindings. :sweat_smile: